### PR TITLE
Move MMI to netstandard2.0 and include OMI v1.6.0

### PR DESCRIPTION
**Pull request by adityapatwardhan**
* Have an Azure DevOps build for CI
* Move to netstandard 2.0
* Move to OMI release v1.6.0
* Create csproj projects for src and test
* Update build and test scripts
* Disable a few tests

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- checkout: self
+  submodules: recursive
+- script: sudo apt-get install libpam0g-dev libssl-dev libcurl4-openssl-dev
+  displayName: 'apt-get dependencies'
+
+- task: ShellScript@2
```
Comment (TravisEz13) [inline azure-pipelines.yml]: Can you use the simple task name?

Comment (TravisEz13) [reply azure-pipelines.yml]: On the other shell task too.

File: test/Microsoft.Management.Infrastructure.Tests/NativeTests/MOFDeserializerTests.cs [hunk 2]
```diff
@@ -23,13 +23,15 @@ public MOFDeserializerTests() : base(MI_SerializationFormat.MOF)
         {
         }
 
+/* @TODO Fix me later
```
Comment (TravisEz13) [inline test/Microsoft.Management.Infrastructure.Tests/NativeTests/MOFDeserializerTests.cs]: file an issue and link

Comment (TravisEz13) [reply test/Microsoft.Management.Infrastructure.Tests/NativeTests/MOFDeserializerTests.cs]: Or maybe list the lines  in the issue

Comment (adityapatwardhan) [reply test/Microsoft.Management.Infrastructure.Tests/NativeTests/MOFDeserializerTests.cs]: https://github.com/PowerShell/MMI/issues/38

File: azure-pipelines.yml (hunk 1 above)
Comment (adityapatwardhan) [reply azure-pipelines.yml]: Fixed

**Pull request merged**